### PR TITLE
Modify kanji generation flow

### PR DESCRIPTION
**Pull request by aruke**
#### Why️?
The existing problem of slow kanji generation could be solved, to some extent.

#### What Changed?

- Use CLI argument in the main script of generator
- Use GitHub Action's matrix strategy to run every kanji generation in parallel
- Also included: For frequency and kanjigarden kanjis, the sheets are now grouped

#### Related issue

NA

### CODE REVIEW

**Comment by aruke** [comment]
#### 📝Changed routes:


| [**/write**](https://kanji-sh-9355ts7xp.vercel.app/write) |
|:-:|
|<a href="https://kanji-sh-9355ts7xp.vercel.app/write">Screenshot of /write</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/X_Ad1WXEyfYvqFMOZiezezwxMec_.png">(view full size)</a> |







Commit d55a0fad4a008f9afc55dc58bc1d44eff8a775bf (https://kanji-sh-9355ts7xp.vercel.app).

File: pages/write/index.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default function WritePage() {
                     <CollectionCard
                         collectionKey={'kanjigarden'}
                         title={'KANJI GARDEN'}
-                        description={'Kanji Garden is a free mnemonic-based SRS kanji learning tool that features about 2900 kanji. '}
+                        description={'Kanji Garden is a free mnemonic-based SRS kanji learning tool that features about 2600 kanji. '}
```
Comment (realmayus) [inline pages/write/index.tsx:42]: After removing all the unnecessary radicals and katakana from the list, 2830 kanji remain

**Review by realmayus — COMMENTED** [review]
I like the changes to the generation flow! It's a lot cleaner :)

File: pages/write/index.tsx (hunk 1 above)
Comment (aruke) [reply pages/write/index.tsx:42]: If you see the latest source file in `generator/sources/kanjigarden/all.source`, there are exactly 2645 kanjis. (I removed extra radicals)

Comment (realmayus) [reply pages/write/index.tsx:42]: Ah, okay then!

**Pull request merged**